src/dotnet/Core/Blobs/Internal/LocalFolderBlobStorage.cs

```diff
@@ -50,8 +56,8 @@ public async Task Write(string path, Stream dataStream, string contentType, Canc
 
         ValidatePath(path);
 
-        var fullPath = _directory & path;
-        Directory.CreateDirectory(fullPath.DirectoryPath);
+        var fullPath = Directory & path;
+        System.IO.Directory.CreateDirectory(fullPath.DirectoryPath);
```

Comment:
Maybe makes sense to use another name in such cases - e.g. RootDirectory, etc.

Reply:
done